src/lib/zip-format.js

```diff
@@ -38,7 +38,7 @@ var commonZipRanges = [
   [210, 601, -5, 1, 391],
   [42001, 42253, -6, 1, 252],
   [42254, 42501, -6, 1, 247],
-  [-1, 210, -4, -1, 211],
+  [-1, 210, -4, 1, 211],
```

Comment:
Do we know why this was -1 before?

Reply:
Don't know. I couldn't find any zip codes between -1 and 210 (exclusive), so I couldn't tie it back to anything factual.  I was thinking it could be some representation of false.  Puerto Rico and the US Virgin Islands are in offset -4 with no DST, but their zip code prefixe are 006 and 008.  By the way, in the zip_codes database table, has_dst is true for PR and the VI, which is incorrect.

Reply:
It's probably a bug from this original findzipranges code which derived this list:

https://github.com/MoveOnOrg/Spoke/commit/b7107ce2c748a7b9e356fdf85fb839788da42148#diff-ea809c7393826e503ba3dbdd94a010f5